### ISSUE TITLE
Text composables: Android Basics with Compose

### ISSUE THREAD

**Issue by TSa3eed** [google-developer-training/basic-android-kotlin-compose-birthday-card-app#486]
**URL of codelab**

https://developer.android.com/codelabs/basic-android-kotlin-compose-text-composables?continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fandroid-basics-compose-unit-1-pathway-3%23codelab-https%3A%2F%2Fdeveloper.android.com%2Fcodelabs%2Fbasic-android-kotlin-compose-text-composables#1

**In which task and step of the codelab can this issue be found?**

[2. Set up a Happy Birthday app] > Create activity ... > Step 2

**Describe the problem**

The instruction say "In the New Project dialog, select Empty Activity and then click Next."
It should specify that the Empty Activity must be an Empty Compose Activity. Otherwise the user can open an Empty Activity making the rest of the steps really hard to follow. 

**Steps to reproduce?**
1. Go to...
2. Click on...
3. See error...

**Versions**
_Android Studio version:_ 
_API version of the emulator:_ 

**Additional information**
_Include screenshots if they would be useful in clarifying the problem._

**Comment by johnshea** [comment]
Hi @TSa3eed 

Thank you for writing in with this issue.

The latest stable version of Android Studio (Currently Giraffe) has them listed as "Empty Activity" and "Empty Views Activity". In prior versions, they were listed as you stated.

As you did not state your Android Studio version, I do not know which one you are using.

If you are on Giraffe and this is still an issue, please create a new ticket and provide a screen shot of the templates shown and also a screenshot of your Android Studio version so we can investigate further.

Hope you are enjoying the course!

Regards.
